static/css/components/header-bar.less

```diff
@@ -48,7 +48,7 @@
       position: absolute;
       z-index: @z-index-level-5;
       text-align: right;
-      width: 125px;
+      width: 179px;
```

Comment:
It should be possible for this menu to work without defining an explicit width.
This width is going to cause us problems later on as we i18n the site.

Can you try using `white-space: nowrap;` on the link instead?

Reply:
@jdlrobson
![image](https://user-images.githubusercontent.com/64412143/104624253-5c281000-56b9-11eb-9042-58bac799d840.png)

Reply:
@jdlrobson
![image](https://user-images.githubusercontent.com/64412143/104624793-04d66f80-56ba-11eb-841b-de884d00afa8.png)